EIPS/eip-6404.md

```diff
@@ -0,0 +1,1000 @@
+---
+eip: 6404
+title: Transition to SSZ
```

Comment:
I think this title could be more descriptive

Reply:
Updated the title to be more specific.